pandas/_libs/hashtable_class_helper.pxi.in

```diff
@@ -1041,7 +1042,8 @@ cdef class StringHashTable(HashTable):
         return np.asarray(locs)
 
     @cython.boundscheck(False)
-    def map_locations(self, ndarray[object] values) -> None:
+    def map_locations(self, ndarray[object] values, object mask = None) -> None:
```

Comment:
not sureif it makes a difference, but you could do `ndarray[uint8_t] mask = None`

Reply:
Wanted to stay consistent with other locations and did not bring an advantage when I tested it